fastlane/actions/podspec_constant_versioning.rb

```diff
@@ -0,0 +1,78 @@
+module Fastlane
+  module Actions
+    class PodspecConstantVersioningAction < Action
+      def self.run(params)
+        podspec = params[:podspec]
+        constant = params[:constant]
+        value = params[:value]
+        
+        podspec_contents = File.read(podspec)
+
+        # TODO: improve RegEx
+        if !podspec_contents.match(/(?<=#{constant})(\s*=\s*["'].*)/)
```

Comment:
1. Consider making the regex a separate variable
1. Regexes can get real hairy, real quick. Consider making it a commented regex with the `/x` flag to aid future devs
1. If you adopt a convention of declaring all versions to be updated in a Podspec preamble, ignore this note. Otherwise: this action appears to be limited to updating versions like `AMPLIFY_VERSION = "1.2.3"`. Are you also going to invoke this for updating the podspec's base `s.version` declaration? If so, will `constant` need to be metacharacter escaped since it includes a dot?